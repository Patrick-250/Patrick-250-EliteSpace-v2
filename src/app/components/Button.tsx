type Props = {
    buttonLabel: string;
}

export default function Button({buttonLabel}: Props) {
    return (
        <button>
            {buttonLabel}
        </button>)
}