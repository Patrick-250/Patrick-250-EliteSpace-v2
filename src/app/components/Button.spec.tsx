import { expect, test } from 'vitest';
import { render } from 'vitest-browser-react'
import Button from './Button';

test('Button', () => {
    const element = render(<Button buttonLabel="test button"/>);
    expect(element).toMatchSnapshot();
})