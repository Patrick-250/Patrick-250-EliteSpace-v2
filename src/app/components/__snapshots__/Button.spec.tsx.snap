// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Button 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    
    
    
  


    <div>
      <button>
        test button
      </button>
    </div>
  </body>,
  "container": <div>
    <button>
      test button
    </button>
  </div>,
  "debug": [Function],
  "getByAltText": [Function],
  "getByLabelText": [Function],
  "getByPlaceholder": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
